.github/workflows/build_daily_dbt_models.yaml

```diff
@@ -4,7 +4,7 @@ on:
   workflow_dispatch:
   schedule:
     # Every day at 8am (2pm UTC-5)
-    - cron: '0 13 * * *'
+    - cron: '0 12 * * *'
```

Comment:
The Python model takes about an hour to build. We could likely speed it up significantly by refactoring it to use spark explicitly. For now, bumping back the daily build by an hour ensure it's ready to go each morning.

Reply:
**[Question, non-blocking]** It seems like [the current run](https://github.com/ccao-data/data-architecture/actions/runs/8925881909/job/24515607821?pr=422) has taken 2.5 hours (so far) to build models, does that interfere at all with this schedule?